proto/proto/test_context.proto

```diff
@@ -4,5 +4,35 @@ import "google/protobuf/timestamp.proto";
 
 package test_context.test_run;
 
+enum TestCaseRunStatus {
+  TEST_CASE_RUN_STATUS_UNSPECIFIED = 0;
+  TEST_CASE_RUN_STATUS_SUCCESS = 1;
+  TEST_CASE_RUN_STATUS_FAILURE = 2;
+  TEST_CASE_RUN_STATUS_SKIPPED = 3;
+}
+
 message TestCaseRun {
+  string id = 1;
+  string name = 2;
+  string classname = 3;
+  string file = 4;
+  // need parent_id
```

Comment:
What's the plan? Populate it on the way out? Add it to the proto?

Reply:
Not sure I understand the question here.

Reply:
Can you rephrase this comment? Is it a todo or is it justifying `parent_name`?